resources/functionMap.php

```diff
@@ -8012,7 +8012,7 @@
 'odbc_do' => ['resource', 'connection_id'=>'resource', 'query'=>'string', 'flags='=>'int'],
 'odbc_error' => ['string', 'connection_id='=>'resource'],
 'odbc_errormsg' => ['string', 'connection_id='=>'resource'],
-'odbc_exec' => ['resource', 'connection_id'=>'resource', 'query'=>'string', 'flags='=>'int'],
+'odbc_exec' => ['resource|false', 'connection_id'=>'resource', 'query'=>'string', 'flags='=>'int'], // Note: 'flags' was removed in PHP 8.0
```

Comment:
The note is useless here, you can submit a change to https://github.com/phpstan/phpstan-src/blob/master/resources/functionMap_php80delta.php instead

Reply:
Okay. Will you do it, or do I need to send a new pull request?

Reply:
Modify this PR so it can be merged, thanks.

Reply:
Ok, I changed the PR